utilities/xmlrpcpp/src/XmlRpcServerConnection.cpp

```diff
@@ -334,8 +344,16 @@ XmlRpcServerConnection::generateResponse(std::string const& resultXml)
   std::string body = RESPONSE_1 + resultXml + RESPONSE_2;
   std::string header = generateHeader(body);
 
-  _response = header + body;
-  XmlRpcUtil::log(5, "XmlRpcServerConnection::generateResponse:\n%s\n", _response.c_str()); 
+  // Avoid an overly large response
+  if ((header.length() + body.length()) > size_t(__INT_MAX__)) {
+    XmlRpcUtil::error("XmlRpcServerConnection::generateResponse: response length (%u) excleeds the maximum allowed size (%u).",
```

Comment:
Small typo `excleeds`

Reply:
Fixed in 7d51a7aaea0cf022cc82357e03db6340741364d3